src/i18n/locales/en/translation.json

```diff
@@ -727,6 +727,11 @@
       "allow_sign": "Allow {{host}} to sign:",
       "block_and_ignore": "Block and ignore {{host}}",
       "block_added": "Added {{host}} to the blocklist, please reload the website."
+    },
+    "transactions": {
+      "title": "All transactions",
+      "description": "This page lists all transactions to websites, independent of the selected account.",
```

Comment:
Only to websites? I ask because I can do transactions using Alby not only to websites, for example paying a lightning invoice.

Reply:
This requirement comes from @bumi he can explain.

Reply:
The way it's implemented is showing "all" transactions though, right? Not only "website" ones. In this case we can just adjust the sentence here and leave out "to websites", right?

Reply:
I think that's correct, all payments. I get the data from https://github.com/lujakob/lightning-browser-extension/blob/master/src/extension/background-script/actions/payments/all.ts#L5-L5

I will remove this part from the translation.

Reply:
I think that's correct, all payments. I get the data from https://github.com/lujakob/lightning-browser-extension/blob/master/src/extension/background-script/actions/payments/all.ts#L5-L5

I will remove this part from the translation.